sandbox-templates/mdx-evaluate-create-react-app/src/App.js

```diff
@@ -0,0 +1,18 @@
+import { evaluateSync } from "@mdx-js/mdx";
+import * as runtime from "react/jsx-runtime";
+
+const mdxContent = `
+# heading
+
+* list
+* item
+
+\`\`\`js
+function () {}
+\`\`\`
+`;
+
+export default function App() {
+  const Content = evaluateSync(mdxContent, { ...runtime }).default;
```

Comment:
I'd prefer pushing async versions when possible (see the new mdx on demand guide on the site for a hook)

Reply:
Sure I can make this async.
the "mdx on demand" guide looks server side rendered? and doesn't appear to be a hook?

Reply:
I meant the way to turn a promise into a value, which is something that guide also does.
So like:

```js
  const [mdxModule, setMdxModule] = useState()
  const Content = mdxModule ? mdxModule.default : Fragment

  useEffect(() => {
    ;(async () => {
      setMdxModule(await run(code, runtime))
    })()
  }, [code])
```

Something like that, but using `evaluate`?

Reply:
updated in f1382be preview at https://codesandbox.io/s/github/ChristianMurphy/.github-mdx/tree/issue-forms/sandbox-templates/mdx-evaluate-create-react-app